url/tests/unit.rs

```diff
@@ -84,8 +84,16 @@ fn new_file_paths() {
     if cfg!(windows) {
         assert_eq!(Url::from_file_path(Path::new("relative")), Err(()));
         assert_eq!(Url::from_file_path(Path::new(r"..\relative")), Err(()));
-        assert_eq!(Url::from_file_path(Path::new(r"\drive-relative")), Err(()));
-        assert_eq!(Url::from_file_path(Path::new(r"\\ucn\")), Err(()));
+        assert_eq!(
+            Url::from_file_path(Path::new(r"\drive-relative")).unwrap(),
+            Url::parse("file:///drive-relative").unwrap()
+        );
+
+        // UNC
+        assert_eq!(
+            Url::from_file_path(Path::new(r"\\server\")).unwrap(),
+            Url::parse("file:///server").unwrap()
```

Comment:
Could you explain again why relative paths would fail to parse, but drive relative paths would not?
In my mind they're not any different.

Reply:
I agree, I just didn't update this part yet (still a draft PR).